charts/fybrik/templates/katalog-connector/katalog-editor-role.yaml

```diff
@@ -16,4 +17,10 @@ rules:
 - apiGroups: ["katalog.fybrik.io"]
   resources: ["assets"]
   verbs: ["get", "list", "watch", "create", "update", "patch", "delete", "deletecollection"]
+{{- if .Values.global.tls.enabled }}
```

Comment:
Shouldn't it be specific to katalog-connector and not according to the global default?

Reply:
I used global settings to simplify configuration. if needed we can change that...